app/models/form/setup/questions/scheme_id.rb

```diff
@@ -24,14 +24,18 @@ def displayed_answer_options(case_log)
 
     user_org_scheme_ids = Scheme.select(:id).where(owning_organisation_id: case_log.created_by.organisation_id).joins(:locations).merge(Location.where("startdate <= ? or startdate IS NULL", Time.zone.today)).map(&:id)
     answer_options.select do |k, _v|
-      user_org_scheme_ids.include?(k.to_i)
+      user_org_scheme_ids.include?(k.to_i) || k.blank?
     end
   end
 
   def hidden_in_check_answers?(case_log, _current_user = nil)
     !supported_housing_selected?(case_log)
   end
 
+  def answer_selected?(case_log, answer)
```

Comment:
Should there be a guard clause here that makes clear this method only applies to select questions?

Reply:
Probably